website/content/docs/concepts/host-discovery/azure.mdx

```diff
@@ -7,6 +7,9 @@ description: >-
 # Azure dynamic host catalogs
 Boundary uses dynamic host catalogs to automatically discover Azure resources available through Azure Resource Manager (ARM) and add them as hosts.
 
+The Azure plugin supports Azure Virtual Machine Scale Sets in both Flexible and Uniform orchestration modes.
```

Comment:
I wouldn't want folks to think we only support VMSS.

`The Azure plugin supports Azure Virtual Machines, and Azure Virtual Machine Scale Sets in both Flexible and Uniform orchestration modes.`

Reply:
Great point, thank you! I updated this line.